src/genericComponents/PrivateLabel.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Badge as PrivateLabel } from 'react-bootstrap';
+import styled from 'styled-components';
+import { colors, fonts } from '../theme/styleVars';
+
+const Badge = () => {
```

Comment:
I know you're importing it as PrivateLabel in the components, which works because it's a default export. However, instead of calling it `Badge` here, let's follow the convention and just call it `PrivateLabel` to start 👍🏻